grr/server/grr_response_server/flows/general/osquery.py

```diff
@@ -19,17 +32,41 @@ class OsqueryFlow(flow_base.FlowBase):
   behaviours = flow_base.BEHAVIOUR_BASIC
 
   args_type = rdf_osquery.OsqueryArgs
+  progress_type = rdf_osquery.OsqueryProgress
+
+  def _UpdateProgress(
+    self,
+    responses: flow_responses.Responses[rdf_osquery.OsqueryResult]
+  ) -> None:
+    # XXX: Less than TRUNCATED_ROWS_COUNT will be used if the original table is
```

Comment:
What is `XXX`?

Reply:
In our codebase the convention is to use `TODO` comments. The format is `TODO(username)` (where `username` is the person that created or should deal with the issue at some point) and `TODO(link)` where `link` is a hyperlink to a bug report or an issue.